addons/stock/models/stock_picking.py

```diff
@@ -565,7 +569,7 @@ def _default_picking_type_id(self):
 
     move_type = fields.Selection([
         ('direct', 'As soon as possible'), ('one', 'When all products are ready')], 'Shipping Policy',
-        default='direct', required=True,
+        compute='_compute_move_type', store=True,
```

Comment:
Imho this should still be required, and be sure to have a fallback in the compute as well (although the Picking Type's `move_type` is required, so I guess we're fine in that regard.

Why I'm saying that is because otherwise, you could set the Picking's `move_type` directly to `False` through the form itself, but the code isn't adapted for that, and will consider it as `one`.

Reply:
Changed :)